core/templates/dev/head/i18n.js

```diff
@@ -75,8 +75,9 @@ oppia.config([
   var availableLanguageKeys = [];
   var availableLanguageKeysMap = {};
   for (var prop in GLOBALS.SUPPORTED_SITE_LANGUAGES) {
-    availableLanguageKeys.push(prop);
-    availableLanguageKeysMap[prop + '*'] = prop;
+    availableLanguageKeys.push(GLOBALS.SUPPORTED_SITE_LANGUAGES[prop].id);
```

Comment:
Actually this part is giving me a lot of problems

```
  GLOBALS.SUPPORTED_SITE_LANGUAGES.forEach(function(language) {
    console.log(language);
  });
```

On running `bash scripts/run_presubmit_checks.sh` this gives an error saying

```
TypeError: Cannot convert undefined or null to object
```

But when I log the value of language it properly shows the objects.

Could you tell me why this is happening?